### PR TITLE
Tila 716 resource api read

### PR DESCRIPTION
**Pull request by aprenfors**

### CODE REVIEW

File: api/graphql/resources/resource_types.py [hunk 1]
```diff
@@ -18,3 +18,9 @@ class Meta:
             "buffer_time_before",
             "buffer_time_after",
         )
+
+        filter_fields = {
+            "name": ["exact", "icontains", "istartswith"],
+        }
+
```
Comment (frwickst) [inline api/graphql/resources/resource_types.py:25]: This change does not correspond to the commit title, and looks like it is adding a filter feature to the code base.

**Review by frwickst — CHANGES_REQUESTED** [review]
Looks good overall 👍 I don't have enough expertise with Graphene to be able to catch everything here, but I can comment on the Git usage a bit :)

The following commits does not conform to the general guidelines for commit naming.
“Descriptive, imperative English briefly describing the change made.” In these cases they don’t full describe what is happening or they do not use an imperative form.

Commits:
137f208603a33b3b7950363ca71f2c2a1fdb8b68
“Resource api end point.”
The issue here is that I don’t know what is happening with the resource api endpoint? Is it changed, added or removed? I’m guessing “add” here?

fdbdb28bf782843fef07cdb6fe2edac9c28db45f
Same the the one above, “what” is happening to the resource read api?

a453856168dd19d77589ade0f022910c7813e244
"Not having errors is better indication than status code in graphql."
The commit message here changes style completely from the rest of the commits. It sounds like a description of why the change is being done, and not the change itself. It also feels as if this is doing more than just “not returning errors” here?

A nitpick here but the term "bracket" generally refers to (, [, {, <. Then generally there are more specific names that are used when talking about one specific bracket. So in the bracket change commit, a more specific title could be "Use round brackets instead of square brackets..." or parenthesis or even "parens".
In this case however, an even more descriptive name (since we know the language we are writing in) could have been "Use tuple instead of lists for field assignment" 😄

File: api/graphql/base_type.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class PrimaryKeyObjectType(DjangoObjectType):
+
+    pk = graphene.Int()
+    class Meta:
+        abstract = True
+
+    def resolve_pk(self, info):
+        return self.id
```
[the remark this answers is not in the document]
Comment (aprenfors) [reply api/graphql/base_type.py]: Everything is formatted automatically with black (and some other tools). There is a newline at end of the file, wonder why this shows it like this, black check fails on missing newline at end so the checks shouldn't even pass without one.

**Comment by frwickst** [comment]
Good work here! Nice changes overall that you made 👍

**Pull request merged**